docs/resources/meeting_conferences.md

```diff
@@ -0,0 +1,526 @@
+variable "app_id" {}
+variable "app_key" {}
+variable "conference_topic" {}
+variable "meeting_room_id" {}
+variable "account_id" {}
+
+resource "huaweicloud_meeting_conference" "test" {
+  app_id  = var.app_id
+  app_key = var.app_key
+
+  topic           = var.conference_topic
+  meeting_room_id = var.meeting_room_id
+  start_time      = "2006-01-02 15:04"
+  duration        = 15
+  media_types     = ["Voice", "Video", "Data"]
+
+  language           = "zh-CN"
+  timezone_id        = 56
+  participant_number = 8
+
+  participants {
+    account_id = var.account_id
+    role       = 1   
+    phone      = "+99123456789876432"
+    email      = "email@example.com"
+  }
+
+  configuration {
+    is_send_notify   = true
+    is_send_calendar = true
+  }
+}
+```
+
+### Book an Ordinary Conference using account authorization
+
+```hcl
+variable "account_name" {}
+variable "account_password" {}
+variable "conference_topic" {}
+variable "meeting_room_id" {}
+variable "account_id" {}
+
+resource "huaweicloud_meeting_conference" "test" {
+  account_name     = var.account_name
+  account_password = var.account_password
+
+  topic           = var.conference_topic
+  meeting_room_id = var.meeting_room_id
+  start_time      = "2006-01-02 15:04"
+  duration        = 15
+  media_types     = ["Voice", "Video", "Data"]
+
+  language           = "zh-CN"
+  timezone_id        = 56
+  participant_number = 8
+
+  participants {
+    account_id = var.account_id
+    role       = 1
+    phone      = "+99123456789876432"
+    email      = "email@example.com"
+  }
+
+  configuration {
+    is_send_notify   = true
+    is_send_calendar = true
+  }
+}
+```
+
+### Book a Cyclical Conference
+
+```hcl
+variable "app_id" {}
+variable "app_key" {}
+variable "conference_topic" {}
+variable "meeting_room_id" {}
+variable "account_id" {}
+
+resource "huaweicloud_meeting_conference" "test" {
+  app_id  = var.app_id
+  app_key = var.app_key
+
+  topic           = var.conference_topic
+  meeting_room_id = var.meeting_room_id
+  duration        = 15
+  media_types     = ["Voice", "Video", "Data"]
+
+  language           = "zh-CN"
+  timezone_id        = 56
+  participant_number = 8
+
+  participants {
+    account_id = var.account_id
+    role       = 1
+    phone      = "+99123456789876432"
+    email      = "email@example.com"
+  }
+
+  cycle_params {
+    cycle      = "Week"
+    pre_remind = 1
+    start_date = "2006-01-02"
+    end_date   = "2006-02-02"
+    interval   = 2
+    points     = [1, 5]
+  }
+
+  configuration {
+    is_send_notify   = true
+    is_send_calendar = true
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `account_name` - (Optional, String, ForceNew) Specifies the (HUAWEI Cloud meeting) user account name to which the
+  meeting initiator belongs. Changing this parameter will create a new resource.
+
+* `account_password` - (Optional, String, ForceNew) Specifies the user password to which the meeting initiator belongs.
```

Comment:
get rid of "to which the meeting initiator belongs" for password.

Reply:
done.